.ado/azure-pipelines.yml

```diff
@@ -111,3 +111,41 @@ jobs:
       - template: templates/apple-yarn-build.yml
         parameters:
           relative_directory: $(relative_directory)
+
+  - job: StaticLibBuild
```

Comment:
Just to be crystal clear- the goal here is to mimic what the publish pipeline does but this NuGet Publish step will be removed after you validate it succeeds right?

Reply:
Yep. I don't want to add this to the PR Pipeline because it takes a while, and we already run `yarn iOS / macOS` which is mostly the same thing. But for PR's where I change the pipeline, I'm trying to be extra careful.